### PR TITLE
2019 standards update

### PR DESCRIPTION
**Pull request by losborne**
Merging update work into master before the gem version change PRs.

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-2134
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated
- [x] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter

**Bonnie Reviewer:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @hackrm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -504,7 +512,7 @@ core-util-is@~1.0.0:
 
 "cql-execution@https://github.com/cqframework/cql-execution#master":
   version "1.3.4"
-  resolved "https://github.com/cqframework/cql-execution#4f9535a7fbcff5923b11ddf7dafe55442c81d1fa"
+  resolved "https://github.com/cqframework/cql-execution#b30335163482b982dfc15dc504a7340a7161b790"
```
Comment (hackrm) [inline yarn.lock]: It seems there have been 2 more commits to cql-execution master branch over the past day or 2 so this may need to be updated.

Comment (losborne) [reply yarn.lock]: Good catch--there will be a separate PR to point to the published cql-execution npm package and update the README--so this is OK for this PR
